WireAuthentication/Sources/WireAuthentication/Components/LoginViaEmailComponent.swift

```diff
@@ -32,24 +32,31 @@ protocol LoginViaEmailComponentDependency: Dependency {
     var networkService: NetworkService { get }
     var environmentType: BackendEnvironmentType { get }
     var backendConfig: BackendConfig { get }
+    var minTLSVersion: TLSVersion { get }
     @MainActor var bridge: WireAuthenticationBridge { get }
 
 }
 
 class LoginViaEmailComponent: Component<LoginViaEmailComponentDependency> {
 
-    public let backendMetadata: WireAuthenticationAPI.BackendMetadata
+    private let environmentType: BackendEnvironmentType
+    private let backendConfig: BackendConfig
```

Comment:
question: do these need to be injected? Can't they be obtained from the parent?

Reply:
because `loginViaEmailComponent` is a child for DetermineAuthMethodComponent, but backendConfig and environmentType runtime parameters. 
I pass them `SwitchBackendConfirmationComponent` -> `DetermineAuthMethodComponent` -> `LoginViaEmailComponent`